README.md

```diff
@@ -59,10 +60,11 @@ See the file [COPYING](COPYING) for more details.
 
 HISTORY
 -------
+XQF was originally written by Roman Pozlevich from Pereslavl-Zalessky, Russia. Over 17 years of existence the project has had the following major developers:
```

Comment:
@illwieckz: Maybe

> XQF was originally written by Roman Pozlevich in 1998. It has been maintained and improved by a devoted team over the years.

Saying "XQF is an active project." in the history section seems out of place to me. I'd either put it in the initial description or (more likely) leave it out. As a user I wouldn't trust that statement to be updated when the project became inactive and look at the commits history to find out with or without the statement.

----

I like the term 'major developers' being used like in skybon's commit.

> XQF was originally written by Roman Pozlevich in 1998. It has been maintained and improved by a devoted team over the years with following major developers: